server/memphis_handlers_configurations.go

```diff
@@ -75,6 +75,65 @@ func (s *Server) initializeConfigurations() {
 	} else {
 		LOGS_RETENTION_IN_DAYS = logsRetention.Value
 	}
+	if configuration.DOCKER_ENV == "" {
+		var brokerHost models.ConfigurationsStringValue
+		err = configurationsCollection.FindOne(context.TODO(), bson.M{"key": "broker_host"}).Decode(&brokerHost)
+		if err != nil {
+			if err != mongo.ErrNoDocuments {
+				s.Errorf("initializeConfigurations: " + err.Error())
```

Comment:
this will continue after logging is it ok?

Reply:
Yes, if an error occurred with mongo, we take a default value and continue